doc/api/deprecations.md

```diff
@@ -3047,6 +3047,20 @@ const w = new Writable({
 });
 ```
 
+### DEP0XXX: Thenable support in streams
```

Comment:
@ronag can you please rebase now that the doc-only deprecation has landed?
```suggestion
### DEP0157: Thenable support in streams
```

Reply:
You need to merge this with https://github.com/nodejs/node/blob/79731df60843a47ed2c4c7b20da773d96157e4f0/doc/api/deprecations.md#L3022.

Reply:
I'm not sure I follow.

Reply:
Instead of adding a new deprecation section, you need to re-use the existing one on line 3022.

Reply:
Here's the diff, maybe that'd be clearer than me trying to find my words:

````diff
diff --git a/doc/api/deprecations.md b/doc/api/deprecations.md
index a543e4851c..5eb1cec12d 100644
--- a/doc/api/deprecations.md
+++ b/doc/api/deprecations.md
@@ -3023,12 +3023,15 @@ it was an aborted or graceful destroy.
 
 <!-- YAML
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/40773
+    description: End-of-life.
   - version: v17.2.0
     pr-url: https://github.com/nodejs/node/pull/40860
     description: Documentation-only deprecation.
 -->
 
-Type: Documentation-only
+Type: End-of-Life
 
 An undocumented feature of Node.js streams was to support thenables in
 implementation methods. This is now deprecated, use callbacks instead and avoid
@@ -3047,20 +3050,6 @@ const w = new Writable({
 });
 ```
 
-### DEP0XXX: Thenable support in streams
-
- <!-- YAML
- changes:
-   - version: REPLACEME
-     pr-url: https://github.com/nodejs/node/pull/40773
-     description: End-of-life.
- -->
-
-Type: End-of-Life
-
-An undocumented feature of Node.js streams was to support thenables in
-implementation methods. This is now deprecated, instead use callbacks.
-
 [Legacy URL API]: url.md#legacy-url-api
 [NIST SP 800-38D]: https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-38d.pdf
 [RFC 6066]: https://tools.ietf.org/html/rfc6066#section-3
````

Reply:
how about now?

Reply:
You need to keep the history, see https://github.com/nodejs/node/pull/40773#discussion_r762551858.